src/main/java/com/giftclub/common/LoginSession.java

```diff
@@ -0,0 +1,31 @@
+package com.giftclub.common;
+
+import com.giftclub.exception.LoginFailedException;
+import com.giftclub.member.Member;
+import com.giftclub.mapper.MemberMapper;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import javax.servlet.http.HttpSession;
+
+@Service
+@RequiredArgsConstructor
+public class LoginSession implements MemberAuthentication {
+
+    private final HttpSession session;
+    private final MemberMapper memberMapper;
+    private final Sha256 sha256;
+
+    @Override
+    public Member loginSession(String memberEmail, String memberPassword) {
+        String sha256Password = sha256.encryption(memberPassword);
+        Member memberInfo = memberMapper.findByEmailAndPassword(memberEmail, sha256Password);
+
+        if(memberInfo == null){

```

Comment:
컨벤션을 지켜주세요~ 다른 곳도 지켜지지 않는 곳이 몇 곳 보이네요~

```suggestion
        if(memberInfo == null) {
```